src/templates/utils/index.ts

```diff
@@ -169,6 +169,10 @@ export const validateTemplateURL = (url): string => {
     return "We can't use that URL"
   }
 
+  if (cleanUrl.includes(' ')) {
+    return 'URL cannot contain spaces'
```

Comment:
the first thing is, i think this would be easier to follow if it was moved to right under where `cleanUrl` is defined. think about the flow of this function. if the template has a space in it, we don't care if it's a community or a correct file type, it's immediately wrong and unparse-able. it's a stronger indicator and so should come first.

the other thing is, the message is kind of mechanical compared to the other messages in this file, which are a little bit more conversational.

maybe something as simple as adding `the` or `your` and dropping the `cannot`

>"Your URL can't contain spaces"